metadata_backend/helpers/schemas/bp_image.json

```
@@ -0,0 +1,307 @@
+                    "title": "Optical Paths",
+                    "type": "object",
+                    "properties": {
+                        "color_space": {
+                            "title": "Color Space",
+                            "type": "string"
+                        },
+                        "color_depth": {
+                            "title": "Color Depth",
+                            "type": "number"
+                        },
+                        "illumination": {
+                            "title": "Illumination",
+                            "type": "object",
+                            "properties": {
+                                "type": {
+                                    "title": "Type",
+                                    "type": "string"
+                                },
+                                "color_code": {
+                                    "title": "Color Code",
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    }
+                },
+                "focus": {
+                    "title": "Focus",
+                    "type": "object",
+                    "properties": {
+                        "focus_method": {
+                            "title": "Focus Method",
+                            "type": "string"
+                        },
+                        "extended_depth_of_field": {
+                            "title": "Extended Depth Of Field",
+                            "type": "boolean"
+                        }
+                    }
+                }
+            }
+        },
+        "base_layer_tile_size": {
+            "title": "Base Layer Tile Size",
+            "type": "object",
+            "properties": {
+                "width": {
+                    "title": "Width",
+                    "type": "number"
+                },
+                "height": {
+                    "title": "Height",
+                    "type": "number"
+                }
+            }
+        },
+        "image_shape": {
+            "title": "Image Shape",
+            "type": "object",
+            "properties": {
+                "width": {
+                    "title": "Width",
+                    "type": "number"
+                },
+                "height": {
+                    "title": "Height",
+                    "type": "number"
+                }
+            }
+        },
+        "studyRef": {
+            "title": "Study Reference",
+            "description": "Identifies the parent study.",
+            "type": "object",
+            "$ref": "#/definitions/reference"
+        },
+        "imageOf": {
+            "title": "Image Of",
+            "description": "One of more samples imaged by the image.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/reference"
+            }
```

This was hot fix, but needs to be handled during parsing. Meaning parser should make this object into array.

```suggestion
            "type": "object",
            "$ref": "#/definitions/reference"
```